standards/qa-process.md

```diff
@@ -111,26 +116,42 @@ a. Labels:
 
   ![project board columns](./assets/qa-process/5.png)
 
-3. When the issue is ready for Retest and deployed to a new test version, the issue will be dragged to “Ready for Retest”. The issue label should be updated from “In Progress” to “Ready for Retest”. QA members will be assigned. Assigned developer will notify the QA team of EVERY PUSH! This is important because a push can work perfectly fine on one end, but break on QA’s end if they haven’t emptied the cache and hard reloaded. 
+3. When the issue is ready for Retest and deployed to a new test version, the issue will be dragged to “Ready for Retest”. The issue label should be updated from “In Progress” to “Ready for Retest”. QA members will be assigned. Assigned developer will notify the QA team of EVERY PUSH! This is important because a push can work perfectly fine on one end, but break on QA’s end if they haven’t emptied the cache and hard reloaded.
 4. If the test is successful, QA Tester adds the “Testing Passed” label to the issue. Issue assigned to POC.
 5. If the testing failed, QA will add feedback on the issue and add the “Testing Failed” label.
-6. Upon completion of the QA Sprint, the POC will close the “Working” issues and “QA Sprint” issue.
+6. If the issue is marked as “Testing Passed”, drag the item to “Done” and close the issue if applicable
+7. Upon completion of the QA Sprint, the POC will close the “Working” issues and “QA Sprint” issue.
+
+## Post Sprint Meeting
+This is an optional meeting for the PM and will be scheduled on the proposed end date with the lead dev if testing is not completed to the satisfaction of definition of done.
+1. QA and dev will review the acceptance test to see what items may have not been marked off.
+2. QA and dev will go over any issues that are still open and issues that need a more thurough explanation if neccessary.
```

Comment:
Typo here:  thorough

Reply:
Ill throw this into another Issue/PR. Thanks!